field_setting/field_setting_parser.py

```diff
@@ -138,17 +125,17 @@ def _parse_codes_comptables(codes_comptables):
 	return fields
 
 
-def _parse_expanse_list(expanse_list):
+def _parse_expense_list(expense_list):
 	fields = dict()
 
-	for i in range(len(expanse_list)):
-		expanse = expanse_list[i]
+	for i in range(len(expense_list)):
+		expense = expense_list[i]
 
-		description = expanse.get("Description")
+		description = expense.get("Description")
 		if description is not None:
```

Comment:
```suggestion
		if (description := expense.get("Description") is not None:
```

Libre à toi d'utiliser le nouvel opérateur d'assignation de Python.